src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String CHECK_LETTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String textFromFile = "";
+        FileReader fileReader;
+        try {
```

Comment:
why do you have two try-catch statements with same exception being caught and same exception being thrown?